Buddies.API/Database/ApiContext.cs

```diff
@@ -6,16 +6,132 @@ namespace Buddies.API.Database;
 
 public class ApiContext : IdentityDbContext<User, Role, int>
 {
+
+    private readonly HttpClient client = new HttpClient();
     public ApiContext(DbContextOptions<ApiContext> options) : base(options)
     {
+        client.DefaultRequestHeaders.Add("user-agent", "Mozilla/4.0 (compatible; MSIE 6.0; Windows NT 5.2; .NET CLR 1.0.3705;)");
+        client.DefaultRequestHeaders.Add("Referer", "http://www.microsoft.com");
```

Comment:
??

Reply:
https://docs.github.com/en/rest/overview/resources-in-the-rest-api#user-agent-required

GitHub requires a valid user agent to make API requests otherwise there will be a forbidden error.